modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2TokenScopeValidatorTestCase.java

```diff
@@ -222,4 +222,38 @@ private TokenIntrospectionResponse introspectAccessToken(AccessToken accessToken
 
         return TokenIntrospectionResponse.parse(introspectionHTTPResp);
     }
+
+    private void createOAuthApplication() throws Exception {
+        restClient = new OAuth2RestClient(serverURL, tenantInfo);
```

Comment:
Shall we add the missing new line here.

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096